app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryDetailOverviewFragment.kt

```diff
@@ -346,4 +376,438 @@ class ContentEntryDetailOverviewFragment: UstadDetailFragment<ContentEntryWithMo
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun ContentDetails(
+    uiState: ContentEntryDetailOverviewUiState = ContentEntryDetailOverviewUiState(),
+){
+    Row {
+
+        Box(
+            modifier = Modifier.weight(0.3F)
+        ) {
+            LeftColumn(
+                uiState = uiState
+            )
+        }
+
+        Spacer(modifier = Modifier.width(10.dp))
+
+        Box(
+            modifier = Modifier.weight(0.7F)
+        ) {
+            RightColumn(
+                uiState = uiState
+            )
+        }
+    }
+}
+
+@Composable
+fun LeftColumn(
+    uiState: ContentEntryDetailOverviewUiState = ContentEntryDetailOverviewUiState(),
+){
+    Column{
+        Image(painter = painterResource(id = R.drawable.book_24px),
+            contentDescription = "",
+            modifier = Modifier.size(110.dp),
+            contentScale = ContentScale.Crop
+        )
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            if (uiState.scoreProgressVisible){
+                LinearProgressIndicator(
+                    progress = 0.6F,
+                    modifier = Modifier
+                        .height(4.dp)
+                        .weight(0.6F),
+                    backgroundColor = contentColorFor(
+                        backgroundColor = colorResource(id = R.color.primaryColor)
+                    )
+                )
+            }
+
+            Spacer(modifier = Modifier.width(10.dp))
+
+            Image(painter = painterResource(id = R.drawable.ic_content_complete),
+                contentDescription = "",
+                modifier = Modifier.size(18.dp)
+            )
+        }
+    }
+}
+
+@Composable
+fun RightColumn(
+    uiState: ContentEntryDetailOverviewUiState = ContentEntryDetailOverviewUiState(),
+){
+    Column(
+        modifier = Modifier.height(IntrinsicSize.Max),
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+
+        Text(
+            text = uiState.contentEntry?.title ?: "",
+            style = Typography.h4
+        )
+
+        if (uiState.authorVisible){
+            Text(text = uiState.contentEntry?.author ?: "")
+        }
+
+        if (uiState.publisherVisible){
+            Text(text = uiState.contentEntry?.publisher ?: "")
+        }
+
+        if (uiState.licenseNameVisible){
+            Row{
+
+                Text(text = stringResource(id = R.string.entry_details_license))
+
+                Spacer(modifier = Modifier.width(5.dp))
+
+                Text(
+                    text = uiState.contentEntry?.licenseName ?: "",
+                    style = Typography.h6
+                )
+            }
+        }
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ){
+
+            if (uiState.fileSizeVisible){
+                Text(text = uiState.contentEntry?.container?.fileSize.toString())
```

Comment:
This needs to use UMFileUtil.formatFileSize . The fileSize on the property is in bytes. formatFileSize will show in KB, MB, etc according to the size.